src/Moryx.Products.Management/Moryx.Products.Management.csproj

```diff
@@ -25,6 +25,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Moryx.Products.Management.xml</DocumentationFile>
+    <LangVersion>7</LangVersion>
```

Comment:
This is part of the global `Directory.build.props`. The current language version is 7.3.